### PR TITLE
Target Size (Minimum) edits and addition of diagram

### PR DESCRIPTION
**Pull request by colabottles**
Hope this is the correct [link to to the latest rendered version](https://raw.githack.com/w3c/wcag/4945bb9ef0e277ea64b9339d832199f005edbcf3/understanding/22/target-size-minimum.html) (1 Nov 2021)

Introduction added as well as the diagram. May need further edits to `<figcaption>` or if there is any additional discussion about the intro text.

Closes https://github.com/w3c/wcag/issues/1848

### CODE REVIEW

File: understanding/22/target-size-minimum.html [hunk 1]
```diff
@@ -72,35 +72,49 @@ <h3>Spacing versus size</h3>
...
+
+        <figure id="target-size-minimum">
+          <img src="img/target-size-minimum-diagram.png"
+            alt="A diagram showing target size of 4 targets, side-by-side that are 24 by 24 pixels in size. Another diagram below the first shows the same targets, reduced in size to 10 by 10 pixels, spaced apart by 14 pixels on all sides showing adjacent button spacing." />
+            <figcaption>
+              Five square targets with dimensions of 24 by 24 pixels, arranged in a cross (one target at the center, and one target above, one to the right, one below, and one to the left), without gaps - this passes. In the second diagram, the central target is shrunk to 10 by 10 pixels, with a margin of 7 pixels to the other targets - this will fail the Success Criterion.
```
Comment (patrickhlauke) [inline understanding/22/target-size-minimum.html:109]: this `<figcaption>` is wrong though ... it describes the other image, not this one here?

File: understanding/22/target-size-minimum.html [hunk 2]
```diff
@@ -72,35 +72,49 @@ <h3>Spacing versus size</h3>
...
+            </figcaption>
+        </figure>
+        <figure id="target-size-minimum-redux">
+          <img src="img/target-size-minimum.png"
+            alt="A diagram showing three targets, two that pass and one that fails Success Criterion." />
+          <figcaption>This shows that to pass, the four outer 24 by 24 pixel targets need to be spaced out further away from the central 10 by 10 pixel target, at a distance of 14 pixels each, making the overall space occupied by the five targets bigger.</figcaption>
```
Comment (patrickhlauke) [inline understanding/22/target-size-minimum.html:115]: i think you swapped the `<figcaption>`s around here by mistake?

**Comment by patrickhlauke** [comment]
Made a few more suggested tweaks to the `alt` and `figcaption` there

**Comment by detlevhfischer** [comment]
Good stuff - just updated the link to the rendere version [above](https://github.com/w3c/wcag/pull/2108#issue-764953251)

**Review by patrickhlauke — APPROVED** [review]

**Comment by patrickhlauke** [comment]
Just on this point (from the survey/meeting about it)

> I wondered about the text on line 103 "...the smaller the actual target is". I think that is potentially not true

Unless I missed something, it is true. For every pixel of width that you make the target smaller, you need to add 1 pixel on both the left spacing and the right spacing. Same for any pixel of height you make it smaller. Effectively, the overall size (or target size and spacing requirement) telescopes out the smaller you make the target size. I thought this made a good point to really try and get authors to consider not straying too far from the ideal size...but fair enough, we can leave it out (it's a bonus discovery for those like me who then run the numbers and try to find the extreme cases) /cc @mbgower

**Pull request merged**